packages/lang/Python2/package.mk

```diff
@@ -1,7 +1,9 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="Python2"
+# When changing PKG_VERSION remember to sync PKG_PYTHON_VERSION!
```

Comment:
Yes, because for Python2 we need to know the PKG_PYTHON_VERSION in numerous Python-related packages when installing stuff for the target, so setting it in `config/functions` avoids sourcing in the Python2 package numerous times just to determine this particular value.

Whereas with Python3, we currently only build it for host, and PKG_PYTHON_VERSION isn't particularly relevant.

Once we drop Python2 we can clean this up a bit.